app/_sass/components/_base.sass

```diff
@@ -10,3 +10,6 @@
 @import separator
 @import code-of-conduct
 @import text
+@import column-container
```

Comment:
like I said previously, this should be renamed to something like ticket-columns. It's not a generic component, the name should indicate so